diff -U3 -r qt-everywhere-opensource-src-5.9.6.orig/qtbase/src/platformsupport/fontdatabases/mac/qfontengine_coretext.mm qt-everywhere-opensource-src-5.9.6/qtbase/src/platformsupport/fontdatabases/mac/qfontengine_coretext.mm
--- qt-everywhere-opensource-src-5.9.6.orig/qtbase/src/platformsupport/fontdatabases/mac/qfontengine_coretext.mm	2018-04-03 21:14:40.000000000 -0700
+++ qt-everywhere-opensource-src-5.9.6/qtbase/src/platformsupport/fontdatabases/mac/qfontengine_coretext.mm	2018-05-07 13:59:20.000000000 -0700
@@ -838,7 +838,7 @@
 
 QFixed QCoreTextFontEngine::emSquareSize() const
 {
-    return QFixed::QFixed(int(CTFontGetUnitsPerEm(ctfont)));
+    return QFixed(int(CTFontGetUnitsPerEm(ctfont)));
 }
 
 QFontEngine *QCoreTextFontEngine::cloneWithSize(qreal pixelSize) const
diff -U3 -r qt-everywhere-opensource-src-5.9.6.orig/qtwebengine/src/3rdparty/chromium/build/mac/should_use_hermetic_xcode.py qt-everywhere-opensource-src-5.9.6/qtwebengine/src/3rdparty/chromium/build/mac/should_use_hermetic_xcode.py
--- qt-everywhere-opensource-src-5.9.6.orig/qtwebengine/src/3rdparty/chromium/build/mac/should_use_hermetic_xcode.py	2018-04-10 07:05:55.000000000 -0700
+++ qt-everywhere-opensource-src-5.9.6/qtwebengine/src/3rdparty/chromium/build/mac/should_use_hermetic_xcode.py	2018-05-07 21:33:42.000000000 -0700
@@ -15,10 +15,11 @@
 
 
 def main():
-  if os.environ.get('FORCE_MAC_TOOLCHAIN') or _IsCorpMachine():
-    return "1"
-  else:
-    return "0"
+  return "0"
+#  if os.environ.get('FORCE_MAC_TOOLCHAIN') or _IsCorpMachine():
+#    return "1"
+#  else:
+#    return "0"
 
 
 if __name__ == '__main__':
diff -U3 -r qt-everywhere-opensource-src-5.9.6.orig/qtwebengine/src/3rdparty/chromium/net/cert/x509_certificate_mac.cc qt-everywhere-opensource-src-5.9.6/qtwebengine/src/3rdparty/chromium/net/cert/x509_certificate_mac.cc
--- qt-everywhere-opensource-src-5.9.6.orig/qtwebengine/src/3rdparty/chromium/net/cert/x509_certificate_mac.cc	2018-04-10 07:05:55.000000000 -0700
+++ qt-everywhere-opensource-src-5.9.6/qtwebengine/src/3rdparty/chromium/net/cert/x509_certificate_mac.cc	2018-05-07 21:56:52.000000000 -0700
@@ -76,7 +76,7 @@
 void GetCertDateForOID(const x509_util::CSSMCachedCertificate& cached_cert,
                        const CSSM_OID* oid,
                        Time* result) {
-  *result = Time::Time();
+  *result = Time();
 
   x509_util::CSSMFieldValue field;
   OSStatus status = cached_cert.GetField(oid, &field);
